helix-core/src/main/java/org/apache/helix/controller/dataproviders/BaseControllerDataProvider.java

```diff
@@ -217,6 +222,7 @@ public String getObjName(StateModelDefinition obj) {
       }
     }, false);
     _currentStateCache = new CurrentStateCache(this);
+    _customizedStateCache = new CustomizedStateCache(this, _aggregationEnabledTypes);
```

Comment:
If you passed "this", I guess you don't need to pass "_aggregationEnabledTypes" in addition. You can always call BaseControllerDataProvider.getCustomizedStateConfig() in the CustomizedStateCache. And that method is safer and cleaner.